packages/inject/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@rollup/plugin-inject",
+  "version": "3.0.2",
```

Comment:
is the peerDep on rollup@^1.20.0 new? If so we need to bump major

Reply:
https://github.com/rollup/rollup-plugin-inject/blob/master/package.json#L9. Technically, we're good. But since that wasn't a peer previously, we should bump